### PR TITLE
Grammatical improvements for the RayCast 2D and 3D class references.

### PR DESCRIPTION
**Pull request by AcatXIo**
Feel free to point out any issues.

### CODE REVIEW

File: doc/classes/RayCast2D.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
 		<method name="force_raycast_update">
 			<return type="void" />
 			<description>
-				Updates the collision information for the ray. Use this method to update the collision information immediately instead of waiting for the next [code]_physics_process[/code] call, for example if the ray or its parent has changed state.
+				Updates the collision information for the ray immediately, without waiting for the next [code]_physics_process[/code] call. The method may be helpful, for example, when the ray or its parent has changed state.
```
Comment (AThousandShips) [inline doc/classes/RayCast2D.xml:39]: The original wording of "Use this method" should be retained, the hypothetical "may be useful" isn't correct for instructions IMO

File: doc/classes/RayCast3D.xml [hunk 2]
```diff
@@ -37,7 +37,7 @@
 		<method name="force_raycast_update">
 			<return type="void" />
 			<description>
-				Updates the collision information for the ray. Use this method to update the collision information immediately instead of waiting for the next [code]_physics_process[/code] call, for example if the ray or its parent has changed state.
+				Updates the collision information for the ray immediately, without waiting for the next [code]_physics_process[/code] call. The method may be helpful, for example, when the ray or its parent has changed state.
```
Comment (AThousandShips) [inline doc/classes/RayCast3D.xml:40]: Same as above

**Review by AThousandShips — APPROVED** [review]
LGTM

File: doc/classes/RayCast2D.xml (hunk 1 above)
Comment (AcatXIo) [reply doc/classes/RayCast2D.xml]: Fixed.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**